### PR TITLE
square of a number

### PR DESCRIPTION
**Pull request by rish9511**

### CODE REVIEW

File: src/activities/rishabh/ActivityInfo.qml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+/* GCompris - ActivityInfo.qml
+ *
+ * Copyright (C) 2015 Your Name <yy@zz.org>
```
Comment (sagaragarwal94) [inline src/activities/rishabh/ActivityInfo.qml]: Small One :) Writing name is necassary.

File: src/activities/rishabh/Rishabh.qml [hunk 2]
```diff
@@ -0,0 +1,168 @@
+/* GCompris - new.qml
+ *
+ * Copyright (C) 2015 YOUR NAME <xx@yy.org>
+ *
+ * Authors:
+ *   <THE GTK VERSION AUTHOR> (GTK+ version)
+ *   YOUR NAME <YOUR EMAIL> (Qt Quick port)
+ *
```
Comment (sagaragarwal94) [inline src/activities/rishabh/Rishabh.qml]: Similar one.

File: src/activities/rishabh/rishabh.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
+/* GCompris - rishabh.js
```
Comment (sagaragarwal94) [inline src/activities/rishabh/rishabh.js]: I think rishabh.js file is not required in this activity. you can remove the unused files.

**Comment by bdoin** [comment]
Thanks for your exercise, stay motivated and focused to bring your work to the quality level expected by GCompris.

**Pull request closed without merging**